build-logic/kotlin/build.gradle.kts

```diff
@@ -8,5 +8,8 @@ dependencies {
     implementation(projects.gradleExt)
     implementation(projects.testing)
     implementation(libs.kotlinGradle)
+    implementation(libs.kotlinx.serialization.gradle)
```

Comment:
Now we have  kotlinX serialization and moshi
could we use only moshi?

Reply:
I'd vote for kotlinx, it's was so much easier for me
Try it!